javascript/javascript3/week1/Giphy_Api/giphy.js

```diff
@@ -0,0 +1,28 @@
+// Resting the page
+
+function sampleFunction() {
+    location.reload();
+}
+
+// getting value from the UI
+function getInputValue() {
+    const gifName = document.getElementById("search-box").value;
+    let limit = 10;
+    limit = document.getElementById("gif-number").value;
+
+    // getting data from api
+
+    fetch(`https://api.giphy.com/v1/gifs/search?q=${gifName}&api_key=I8lYnqjSwMSOegwaI6CM3wXi88fVqwCR&q=&limit=${limit}&offset=0&rating=g&lang=en`)
+        .then(response => response.json())
+        .then(gifData => {
+            console.log(gifData);
+            let gifImag = gifData.data
+            gifImag.forEach(gif => {
+                console.log(gif.url);
```

Comment:
You can use `console.log` while implementing, but remember to remove them before you commit your changes